pae-examples.xhtml

```diff
@@ -137,13 +137,13 @@ version-footer: true
                 <div class="highlight"><pre><code id="code_pae4"></code></pre></div>
             </p>
 
-            <h4>Grace notes (accaciature)</h4>
+            <h4>Grace notes (accacciature)</h4>
             <div id="svg_pae5" style="overflow:auto;"/>
             <p>
                 <div class="highlight"><pre><code id="code_pae5"></code></pre></div>
             </p>
 
-            <h4>Grace notes (appogiature)</h4>
+            <h4>Grace notes (appoggiature)</h4>
```

Comment:
Similar to acciaccature/acciaccaturas, a more preferred plural in English is appoggiaturas:
     https://en.wiktionary.org/wiki/appoggiatura